src/components/map/Map.js

```diff
@@ -122,14 +122,10 @@ class Map extends Component {
 
     render() {
         const { basemap, layers, openContextMenu, classes } = this.props;
-        const overlays = [...layers.filter(layer => layer.isLoaded)].reverse();
```

Comment:
Does this not need to be reversed anymore?

Reply:
I did an extra check, and pushed a small fix. It is now rendering the layers correctly in both the app and the plugin. The layers are still reversed here: https://github.com/dhis2/maps-app/blob/feat/timeline/src/components/map/MapView.js#L40